src/components/BookForm.js

```diff
@@ -0,0 +1,25 @@
+const BookForm = () => {
+  const bookCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+  return (
+    <form>
+      <div>
+        <label htmlFor="title">
+          Title of your book:
```

Comment:
- [ ] Please move BookForm to src/container. Because it qualifies as a container.